containers/geth/syncing-blockchain.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/sh -e
+
+response="$(
+    wget                                                                            \
+        -O-                                                                         \
+        --quiet                                                                     \
+        --post-data='{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":4}'   \
+        --header="Content-Type:application/json"                                    \
+        "geth.default.svc.cluster.local:8545"                                                            \
+        | jq '.result'                                                              \
+)"
+
+if [[ "$response" == "false" ]]; then
+    echo READY
+    exit 0
+fi
```

Comment:
What if `$response` is something else than `true` or `false`?

Currently the script will treat is as `true` which is not a good idea in my opinion. If we get something unexpected, it's likely to be an error. Maybe the best option would be to return 2 and print an error message to stderr, saying that the response was unexpected.